src/sentry/integrations/issues.py

```diff
@@ -235,6 +235,26 @@ def create_default_repo_choice(self, default_repo):
         """
         return (default_repo, default_repo)
 
+    def get_annotations(self, group):
+        external_issue_ids = GroupLink.objects.filter(
+            group_id=group.id,
+            project_id=group.project_id,
+            linked_type=GroupLink.LinkedType.issue,
+            relationship=GroupLink.Relationship.references,
+        ).values_list('linked_id', flat=True)
+
+        external_issues = ExternalIssue.objects.filter(
+            id__in=external_issue_ids,
+            integration_id=self.model.id,
+        )
+        annotations = []
+        for ei in external_issues:
+            link = self.get_issue_url(ei.key)
+            label = self.get_issue_display_name(ei) or ei.key
+            annotations.append('<a href="%s">%s</a>' % (link, label))
```

Comment:
Should we consider not shipping HTML to the client an send a dictionary of `url` and `label` that we can turn into HTML?

Reply:
Maybe long term, but wanted to at least make it work with https://github.com/getsentry/sentry/blob/f1dc069482c36d738ac222ace4412274dd650fff/src/sentry/static/sentry/app/components/eventOrGroupExtraDetails.jsx#L115